src/XRP/serializer.ts

```diff
@@ -135,6 +147,23 @@ const serializer = {
   xrpAmountToJSON(xrpDropsAmount: XRPDropsAmount): string {
     return `${xrpDropsAmount.getDrops()}`
   },
+
+  /**
+   * Convert a Memo to a JSON representation.
+   * @param memo - The Memo to convert.
+   * @returns The Memo as JSON.
+   */
+  memoToJSON(memo: Memo): MemoJSON {
+    const Memo: MemoDetailsJSON = {
```

Comment:
don't uppercase your variable names

Reply:
I have a change for this already that I haven't committed. The reason I did this was because the variable names are upper cased in the actual spec and I was trying to reference it by name alone in the associated object.